redirects.yaml

```
@@ -717,5 +717,6 @@ certification(?P<page>(/.*)?): "https://certification.ubuntu.com{page}"
 usn/?: "https://usn.ubuntu.com/"
 usn/rss\.xml: "https://usn.ubuntu.com/rss.xml"
 (?P<page>.*)\.html: "/{page}"
+(?P<page>.+?)\/: "/{page}"
```

I'm not quite sure what the `?` is doing in this expression. Is `.+?` the same as `.*`? If not, what does it mean? I also don't think `/` needs to be escaped.

I think this should work:

```suggestion
(?P<page>.+)/: "/{page}"
```

---

Works! Thank you